### PR TITLE
Make GetPathRelativeTo() resolve symbolic links

### PR DESCRIPTION
**Pull request by infraredgirl**
Related to #75.

Without this change the relative paths are not computed correctly in case when `basePath` is a subfolder of `path`, and one  of them is a symbolic link of the other.

Note that I had to create fixtures for test cases of `GetPathRelativeTo()`, because the `filepath.EvalSymlinks()` method that we're now using is actually looking for files on disk.

### CODE REVIEW

File: files/paths.go [hunk 1]
```diff
@@ -77,12 +77,22 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 		basePath = "."
```
Comment (zackproser) [inline files/paths.go:77]: NIT: Is the comment above this function now out of date, or should it be updated to mention it evaluates symbolic links?

**Review by zackproser — DISMISSED** [review]
LGTM

File: files/paths.go (hunk 1 above)
Comment (infraredgirl) [reply files/paths.go:78]: Good point - fixed in ddb2085!

**Review by zackproser — APPROVED** [review]
LGTM!

**Comment by infraredgirl** [comment]
Thanks for the review, Zack! Merging.

**Pull request merged**